npc/custom/quests/questboard.txt

```diff
@@ -93,7 +94,7 @@ prontera,129,215,5	script	Questboard	4_BOARD3,{
 			if(gettimetick(2) < getd(.@selection + "_collection_delay")){
 				set .@time_left, getd(.@selection + "_collection_delay")-gettimetick(2);
```

Comment:
Please simply change this and remove the substraction of the gettimetick here.

Reply:
I don't know what you mean, but the changes that I gave resolve this problem :)

Reply:
`.@time_left` is a scope variable. It is followed by a close, and isn't used elsewhere, so it can be safely removed.